realtime.go

```diff
@@ -99,16 +99,37 @@ type StopTimeEvent struct {
 }
 
 type VehicleID struct {
-	ID           string
-	Label        string
-	LicencePlate string
+	ID           *string
+	Label        *string
+	LicencePlate *string
 }
 
+type Position = gtfsrt.Position
+type CurrentStatus = gtfsrt.VehiclePosition_VehicleStopStatus
+type CongestionLevel = gtfsrt.VehiclePosition_CongestionLevel
+type OccupancyStatus = gtfsrt.VehiclePosition_OccupancyStatus
+
 type Vehicle struct {
 	ID *VehicleID
 
 	Trip *Trip
 
+	Position *Position
+
+	CurrentStopSequence *uint32
+
+	StopID *string
+
+	CurrentStatus *CurrentStatus
+
+	Timestamp *time.Time
+
+	CongestionLevel *CongestionLevel
```

Comment:
Would it be possible to make this not a pointer, and if the field is unset then set it to `UNKNOWN_CONGESTION_LEVEL`?